lib/order.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :sum, :basket, :menu
+
+  def initialize(order_menu)
+    @menu = order_menu.list
+    @basket = Hash.new(0)
+    @sum = 0
+  end
+
+  def add(dish, quantity)
+    if @menu.has_key?(dish.to_sym)
```

Comment:
See comment above about adding methods to `Menu` to deal with these types of requests.